src/components/image/image.tsx

```diff
@@ -126,18 +113,12 @@ export class Image
    */
   @Prop({ mutable: true }) width: string;
 
-  /**
-   * Emitted when the element is clicked.
-   */
-  @Event() gxClick: EventEmitter;
-
-  private handleClick(event: UIEvent) {
+  @Listen("click", { capture: true })
+  handleClick(event: UIEvent) {
```

Comment:
Can we keep this method private?

Reply:
it isn't private... it's a handle for the event assigned ("clicked").
Can it be the handler and be private at same time?

Reply:
Yes, it can.

Reply:
If I set the handler as  `private`, it takes that as a method...